doc/source/data/preprocessors.rst

```diff
@@ -55,77 +47,14 @@ Finally, call ``transform_batch`` on a single batch of data.
     :start-after: __preprocessor_transform_batch_start__
     :end-before: __preprocessor_transform_batch_end__
 
-Life of an AIR preprocessor
----------------------------
-
-Now that we've gone over the basics, let's dive into how ``Preprocessor``\s fit into an end-to-end application built with AIR.
-The diagram below depicts an overview of the main steps of a ``Preprocessor``:
-
-#. Passed into a ``Trainer`` to ``fit`` and ``transform`` input ``Dataset``\s
-#. Saved as a ``Checkpoint``
-#. Reconstructed in a ``Predictor`` to ``fit_batch`` on batches of data
-
-.. figure:: images/air-preprocessor.svg
-
-Throughout this section we'll go through this workflow in more detail, with code examples using XGBoost.
-The same logic is applicable to other machine learning framework integrations as well.
-
-Trainer
-~~~~~~~
-
-The journey of the ``Preprocessor`` starts with the :class:`Trainer <ray.train.trainer.BaseTrainer>`.
-If the ``Trainer`` is instantiated with a ``Preprocessor``, then the following logic is executed when ``Trainer.fit()`` is called:
-
-#. If a ``"train"`` ``Dataset`` is passed in, then the ``Preprocessor`` calls ``fit()`` on it.
-#. The ``Preprocessor`` then calls ``transform()`` on all ``Dataset``\s, including the ``"train"`` ``Dataset``.
-#. The ``Trainer`` then performs training on the preprocessed ``Dataset``\s.
+The most common way of using a preprocessor is by using it in conjunction with a Ray Train :ref:`Trainer <train-docs>` and a :ref:`Ray Data dataset <data>`.
```

Comment:
with the updated code snippet, the preprocessor is not actually used with the Trainer.

Reply:
Updated verbiage